fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/physical/PhysicalSetOperation.java

```diff
@@ -173,9 +173,9 @@ public boolean pushDownRuntimeFilter(CascadesContext context, IdGenerator<Runtim
                 if (projIndex < 0 || projIndex >= project.getProjects().size()) {
                     continue;
                 }
-                NamedExpression newProbeExpr = project.getProjects().get(projIndex);
+                Expression newProbeExpr = project.getProjects().get(projIndex);
```

Comment:
newProbeExpr is NamedExpression. please revert this line

Reply:
no, newProbeExpr could be other expression, so must change to Expression